rootfs/api/models/app.py

```diff
@@ -329,7 +329,7 @@ def _scale_pods(self, scale_types):
                 log_event(self, err, logging.ERROR)
                 raise
 
-    def deploy(self, user, release):
+    def deploy(self, release):
```

Comment:
They can be a separate PR. It was mostly that I didn't want to add in a fake user or use a random user when calling deploy from the management script